.github/workflows/release.yml

```diff
@@ -48,6 +53,13 @@ jobs:
           toolchain: nightly-2023-02-14
           targets: ${{ matrix.job.target }}
 
+      - uses: Swatinem/rust-cache@v2
```

Comment:
This Action makes a lot easier to cache if we are only dealing with a Rust project. Since the precompilation is entirely done by cargo, and we have control over the place the target is saved, I decided to use this here.

Reply:
Another thing I didn't mention is that this caching should reduce the precompilation time in half, which is great! :rocket: